model_analyzer/config/input/config_command.py

```diff
@@ -96,6 +97,11 @@ def set_config_values(self, args):
         else:
             yaml_config = None
 
+        validator = YamlConfigValidator()
+        if yaml_config:
+            for option in yaml_config.keys():
+                validator.is_valid_option(option)
```

Comment:
this function returns a bool, but you don't do anything with it?
will logger.error actually error out? I think not? Maybe keep a flag (`failed |= validator.is_valid_option()`), and then after: `if failed: raise exception`

Reply:
done